<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDesigner</class>
 <widget class="QMainWindow" name="ModelDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zing Model Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnewproject"/>
    <addaction name="actionnewstm"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionfilemanager"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionPowerlink"/>
    <addaction name="actionModbus"/>
    <addaction name="actionTCP_IP_Simplify"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionnewstm">
   <property name="text">
    <string>New STM(&amp;S)</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>Open(Ctrl+O)</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>Save(Ctrl+S)</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionignore">
   <property name="text">
    <string>忽略单元格</string>
   </property>
  </action>
  <action name="actionnormal">
   <property name="text">
    <string>常规单元格</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionfilemanager">
   <property name="text">
    <string>Resource browser</string>
   </property>
  </action>
  <action name="actionnewproject">
   <property name="text">
    <string>New Project(&amp;N)</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>Version information</string>
   </property>
  </action>
  <action name="actionPowerlink">
   <property name="text">
    <string>PowerLink</string>
   </property>
  </action>
  <action name="actionModbus">
   <property name="text">
    <string>ModBus</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionTCP_IP_Simplify">
   <property name="text">
    <string>TCP/IP</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
